poglink/cogs/rates.py

```diff
@@ -186,6 +187,14 @@ async def compare_and_notify_all(self):
     async def on_ready(self):
         logger.info("Cog Ready: Rates")
 
+        # publish initial rates for each url upon startup.
+        if self.publish_on_startup:
+            blank_rates = RatesStatus()
+            for url in self.webpage_urls:
+                rates = self.get_current_rates(url)
+                rates_diff = rates.get_diff(blank_rates)
```

Comment:
Just comparing current rates against an empty `RatesStatus` object to get a diff, which can then be converted to a properly formatted embed.

This should bold everything, but if you don't want it bolded, you could `get_diff()` against itself instead.

Also if you want the embed to look different on this initial publish, then you could just write any other method of generating that embed description.